### ISSUE TITLE
False positive with missing semicolon

### ISSUE THREAD

**Issue by aleksey-bykov** [palantir/tslint#364]
```ts
export function plantBlock<a>(
    beDisposed: ed.Disposables,
    section: Section<a>,
    settings: ei.TicksSettings<a>,
    scale: R<Px, a>,
    containerElement: Element,
    formatValue: (value: N<a>) => string
) : UnfixedBlock<a> | Block<a> {
 
    var blockElement = d3.select(containerElement).append('g').node(); // <-- missing semicolon
```

```ts
export function plantBlock<a>(
    beDisposed: ed.Disposables,
    section: Section<a>,
    settings: ei.TicksSettings<a>,
    scale: R<Px, a>,
    containerElement: Element,
    formatValue: (value: N<a>) => string
) {
 
    var blockElement = d3.select(containerElement).append('g').node(); // <-- no problem
```

**Comment by leeavital** [comment]
What rules are you using? I can't reproduce this using only the semicolon rule.


My tslint.json:
````json
{
  "rules": {
    "semicolon": true
  }
}
````

**Comment by adidahiya** [comment]
closing due to inactivity
